docs/configuration.md

```diff
@@ -60,15 +60,18 @@ Property
+Property                                                                                                                                           | Description                                                                                                                                                                               | Default value
+-------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------
+**envoy-control.envoy.snapshot.incoming-permissions.enabled**                                                                                      | Enable incoming permissions                                                                                                                                                               | false
+**envoy-control.envoy.snapshot.incoming-permissions.client-identity-header**                                                                       | Header that identifies the client that called the endpoint                                                                                                                                | x-service-name
+**envoy-control.envoy.snapshot.incoming-permissions.source-ip-authentication.ip-from-service-discovery.enabled-for-incoming-services**             | Enable source ip based authentication for selected services                                                                                                                               | empty list
+**envoy-control.envoy.snapshot.incoming-permissions.source-ip-authentication.ip-from-range.ip-range-for-services**                                 | Enable source ip based authentication for static ranges                                                                                                                                   | empty map
+**envoy-control.envoy.snapshot.incoming-permissions.source-ip-authentication.ip-from-range.ip-range-for-services.{name}**                          | User chosen name for static ip range, it must match client name defined in incoming permissions                                                                                           | none
+**envoy-control.envoy.snapshot.incoming-permissions.source-ip-authentication.ip-from-range.ip-range-for-services.{name}.{ip}/{prefix}**            | Ip and prefix informat of {ip}/{prefix} for static ip range. Eg: 192.168.1.0/24                                                                                                           | empty string
```

Comment:
The properties are invalid. They have `ip-range-for-services` segment which is not present in actual properties.